Utilities/rANS/test/test_ransFrequencyTable.cxx

```diff
@@ -0,0 +1,80 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   test_ransFrequencyTable.cxx
+/// @author Michael Lettrich
+/// @since  Aug 1, 2020
+/// @brief
+
+#define BOOST_TEST_MODULE Utility test
+#define BOOST_TEST_MAIN
+#define BOOST_TEST_DYN_LINK
+
+#include <boost/test/unit_test.hpp>
+
+#include "rANS/rans.h"
+
+BOOST_AUTO_TEST_CASE(test_addSamples)
```

Comment:
@shahor02 see the tests on how samples or whole histograms can be appended, mixed and merged.